docs/pyproject.toml

```diff
@@ -0,0 +1,19 @@
+[tool.poetry]
+name = "parsec-docs"
+version = "0.0.0"
+description = "The parsec-cloud documentations"
+authors = ["Scille SAS <contact@scille.fr>"]
+license = "BUSL-1.1"
+readme = "../README.md"
+packages = []
+
+[tool.poetry.dependencies]
+python = "^3.9"
```

Comment:
Should this file also be added to the `/misc/version_updater.py` script? (it contains a Python version)